### DISCUSSION TITLE
Can't import pyopenvdb in Windows 10

### DISCUSSION THREAD

**Discussion by diegogangl** [AcademySoftwareFoundation/openvdb#1126] [Q&A]
(follow up from issue #1125 )

**Operating System**: Windows 10 / Python 3.9
**Version / Commit SHA**: VDB 8.0.1
**CMake Version**: 3.20.4
**Compiler**: msvc 2019

I managed to build the OpenVDB for Python, but importing always fails due to some missing dll.

I've tried:
- Copying DLLs into the folder (`Half-2_5.dll`, `libzstd.dll`, `openvdb.dll` and `boost_python39-vc142-mt-gd-x64-1_76.dll`). I got these from the Dependencies tool.
- Also tried adding `snappy_64.dll`
- Appending the path to `sys.path`
- Adding the path with `os.add_dll_directory()`
- Look around the source code to figure out which dll is trying to find

No matter what I try I always get

```python
>>> import pyopenvdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: DLL load failed while importing pyopenvdb: The specified module could not be found.
```

I'm trying to figure out what's missing with the dependency walker app, but it doesn't seem like these would be a problem? I've installed the Microsoft Visual C ++ redistributable 2015-2019 for both x86 and 64 but the DLLs are still missing.
![deps](https://user-images.githubusercontent.com/264829/125839566-31e4fed5-fc1b-4838-b00c-513b489eb47a.jpg)

**Comment by 3dfernando** [comment]
4 years later and still doesn't support windows??

**Comment by mimi99528** [comment]
Same problem.
